packages/headless/src/features/commerce/common/actions.test.ts

```diff
@@ -189,6 +190,16 @@ describe('commerce common actions', () => {
       });
     });
 
+    it('sets the capture property from the analytics configuration', async () => {
+      const request = await Actions.buildCommerceAPIRequest(state);
+
+      expect(mockedBuildBaseCommerceAPIRequest).toHaveBeenCalledWith(state);
+
+      expect(request.context.capture).toEqual(
```

Comment:
Since it defaults to true in the initial state, I think we should test that it's correctly updated if you set it to false.

Reply:
Done in https://github.com/coveo/ui-kit/pull/4057/commits/23d27b5c73e0db89db75dd6a021ee038f4eb9e1b